src/Symfony/Component/Mailer/Command/MailerDebugCommand.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+/**
+ * Helps making sure your mailer provider is operational.
+ *
+ * @author Guillaume MOREL <me@gmorel.io>
+ *
+ * @final
+ */
+class MailerDebugCommand extends Command
+{
+    /** {@inheritdoc} */
+    protected static $defaultName = 'debug:mailer:email:send';
+
+    /** @var MailerInterface */
+    private $mailer;
+
+    /** @var SymfonyStyle */
+    private $io;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(MailerInterface $mailer)
+    {
+        parent::__construct();
+
+        $this->mailer = $mailer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setDescription('Send simple email message')
+            ->setDefinition([
+                new InputOption('from', null, InputOption::VALUE_REQUIRED, 'The from email of the message'),
+                new InputOption('to', null, InputOption::VALUE_REQUIRED, 'The to email of the message'),
+                new InputOption('subject', null, InputOption::VALUE_REQUIRED, 'The subject of the message'),
+                new InputOption('body', null, InputOption::VALUE_REQUIRED, 'The body of the message'),
+            ])
+            ->setHelp(
+                <<<EOF
+The <info>%command.name%</info> command creates and sends a simple email message.
+Usage:
+- <info>php %command.full_name%</info> will trigger the interactive mode
+- <info>php %command.full_name% --from=a@symfony.com --to=b@symfony.com --subject=Test --body=body</info>
```

Comment:
passing the body as an option is probably unusable. The SwiftmailerBundle command was supporting passing a file name or using stdin for the body.

Reply:
The purpose was only to quickly test. But I can indeed implement also the passing file args :+1:

Reply:
Passing an optional file would be a nice one 👌🏻